packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -3,6 +3,7 @@
 
 import * as fs from "fs";
 import * as jszip from "jszip";
+import { startDetachedProcess } from "office-addin-debugging";
```

Comment:
Is this the only reason for office-addin-dev-settings to use office-addin-debugging?  We might want to discuss moving the process/port stuff out of office-addin-debugging to a "lower level' package.

Reply:
Yes so I can use startDetachedProcess.  I can work around this by creating a duplicate instance of this method in office-addin-dev-settings, but obviously I would rather not have to do this

Reply:
Rather than copy it, you can just move process.ts from office-addin-debugging to office-addin-dev-settings, then have office-addin-debugging import it (and also re-export it for backward compatibility).

Reply:
Moved process.ts functions to office-addin-dev-settings, and then have existing process.ts in office-addin-debugging import those functions and call them for backwards compat